docs/state-store-overview.md

```diff
@@ -34,6 +42,38 @@ Streaming state store has distinguished workload characteristics.
 
 This leads to the design of Hummock, the cloud-native KV-based streaming state store. We’ll explain concepts like “epoch”, “key space” and “barrier” in the following chapters.
 
+## Cell-based Relational Table
+[source code](https://github.com/singularity-data/risingwave/blob/main/src/storage/src/table/state_table.rs)
```

Comment:
I feel like it's a doc on state table instead of Cell-based Relational Table. For Cell-based Relational Table, may be we should focus on the how it is encoded in storage.

Reply:
`Cell-based` is just an encoding mode in relational table, and perhaps there will be more encoding like `Row-based` or `Column-based`. 
The purpose of this doc is to explain the relational layer between stateful executors and state store, so how about remove `cell-based`?